spopt/locate/p_median.py

```diff
@@ -547,7 +547,7 @@ def solve(self, solver: pulp.LpSolver, results: bool = True):
 class KNearestPMedian(PMedian):
     r"""
     Implement the P-Median Model with Near-Far Cost Allocation and solve it.
-    The model is adapted from :cite:`richard_2018`, can be formulated as:
+    The model is adapted from :cite:`richard_2018`, and can be formulated as:
```

Comment:
This should probably be keyed "church_2018", no?

Reply:
Anyway, I don't mind if it's not.

Reply:
Looks like it snuck in as [`richard_2018 `](https://github.com/pysal/spopt/blob/a330389c767a883600af8cb8b76fca8d63610c3f/docs/_static/references.bib#L171).

Reply:
I'll throw in a quick PR for a fix.